.changeset/seven-badgers-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+This adds a configuration option to the scaffolder plugin router, so we can allow for multiple taskworkers. Currently with only one taskworker you are limited to scaffolding one thing at a time. Set the `taskWorkers?: number` option in your scaffolder router to get more than 1 taskworker
```

Comment:
`TaskWorker` or `TaskWorkers` is probably better to use here. then you can remove the `vocab.txt` entries I think as they're codeblocks.